src/v2-router.js

```diff
@@ -232,6 +236,38 @@ const cleanRevokeInput = (existingId, body) => ({
   isRevoked: body.isRevoked
 });
 
+/**
+ * Build a JWT to allow a visitor to log in to the renewal flow.
+ *
+ * @param {string} jwtPrivateKey
+ * @param {string} email
+ * @returns {string} a signed JWT
+ */
+const buildRenewalToken = (jwtPrivateKey, email) =>
+  jwt.sign({}, jwtPrivateKey, {subject: `${email}`, algorithm: 'ES256', expiresIn: '30m', noTimestamp: true});
+
+/**
+ * Send an email to the visitor that contains a link which allows them to log in
+ * to the rest of the meat bait return system.
+ *
+ * @param {string} notifyApiKey API key for sending emails
+ * @param {string} emailAddress where to send the log in email
+ * @param {string} loginLink link to log in via
+ * @param {string} regNo trap registration number for notify's records
+ */
+const sendRenewalEmail = async (notifyApiKey, emailAddress, loginLink) => {
+  if (notifyApiKey) {
+    const notifyClient = new NotifyClient.NotifyClient(notifyApiKey);
+
+    await notifyClient.sendEmail('173ce02f-b78f-44eb-ac72-2a58af3606a9', emailAddress, {
```

Comment:
This template requires a `Name` field in it's personalisation object.

Reply:
Resolved in https://github.com/Scottish-Natural-Heritage/trap-registration-api/pull/124/commits/5c55bc20ba9e89291e7dcc8936715a64d6809c49.